boilerplate/fullstack/next-app-dir/app/components/home.tsx

```diff
@@ -7,39 +7,82 @@ import { CelebrateIcon, SeparatorLine } from "../../assets/images";
 import { CallAPIButton } from "./callApiButton";
 import { LinksComponent } from "./linksComponent";
 import { SessionAuthForNextJS } from "./sessionAuthForNextJS";
-import { getSSRSession } from "supertokens-node/nextjs";
-import { SessionContainer } from "supertokens-node/recipe/session";
 import { ensureSuperTokensInit } from "../config/backend";
```

Comment:
there is also nextjs pages dir that needs to change in the getServerSideProps function

Reply:
Yeah, that is only in the docs. I will be updating that in the docs PR.